### PR TITLE
feature(Parser): detect if original source have elevation data.

### PR DESCRIPTION
**Pull request by gchoqueux**
## Description
Parser detect if original source have elevation data.

## Motivation and Context
After parsering, there is no way to have this information.
We will need it for the label refactoring

### CODE REVIEW

File: src/Core/Feature.js [hunk 1]
```diff
@@ -223,6 +223,7 @@ function push3DValues(value0, value1, value2 = 0) {
  * @property {number[]} normals - All the normals of the Feature.
  * @property {number} size - the number of values of the array that should be associated with a coordinates.
  * The size is 3 with altitude and 2 without altitude.
+ * @property {boolean} hasRawElevationData - indicates if the raw data,from original source, has elevation.
```
Comment (jailln) [inline src/Core/Feature.js]: I would add `(i.e. is 3D)` at the end of the comment to be even clearer

Comment (gchoqueux) [reply src/Core/Feature.js]: or a formulation more sig
`the geographic coordinates, from original source, have an elevation`

Comment (jailln) [reply src/Core/Feature.js]: yes why not but I would still add explicitly something like "is 3D" or "has a third coordinate"

Comment (gchoqueux) [reply src/Core/Feature.js]: ok 👍🏽

**Review by jailln — APPROVED** [review]

**Pull request merged**